UI.tsv

```diff
@@ -459,10 +459,10 @@ UI_20150317_000458	{@st41b}Create Party
 UI_20150317_000459	{@st43}Public Party List {/}
 UI_20150317_000460	${@st59b}Public Party List{nl}{@st59s} - Search for parties you want to join. {nl} - Ticking "Use Search" uses your search conditions to help find desirable parties. {/}
 UI_20150317_000461	{@st59}Close Party List{/}
-UI_20150317_000462	Use Search
+UI_20150317_000462	Use the Search Feature
 UI_20150317_000463	Quest Sharing
-UI_20150317_000464	EXP Priority
-UI_20150317_000465	Item Priority
+UI_20150317_000464	EXP Distribution
+UI_20150317_000465	Item Distribution
```

Comment:
Agree with Grillo. Distribution is more intuitive, unless the system gave party leaders a way of ranking party members from first to last priority.